### PR TITLE
Closes #2570: `bigint_agg_test.py` Conversion for new test framework

### LINKED ISSUE #2570: `bigint_agg_test.py` Conversion for new test framework
Convert `bigint_agg_test.py` to work in the new test framework. The existing file should be maintained since we will be doing a complete swap once all tests are converted. Only the existing tests need to be converted, there should not be any additional testing that needs to be added.

### PR DESCRIPTION
**Pull request by TheOddCzar789**
This PR (Closes #2570):
  - Converts file to new test framework by adding pytest

### CODE REVIEW

**Review by jaketrookman — CHANGES_REQUESTED** [review]
It might be worth making `SIZE` an array of values and parameterizing it to allow for easier updates for future implementation.

File: PROTO_tests/tests/bigint_agg_test.py
Comment (pierce314159) [inline PROTO_tests/tests/bigint_agg_test.py:1]: I agree with @jaketrookman that we should use `pytest.prob_size` as a parameterization for `SIZE`

File: PROTO_tests/tests/bigint_agg_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    def test_change_size(self, size):
+        # Go from 256 bigint values down to just 1
+        bits = ak.randint(0, 2**32, size, dtype=ak.uint64)
+        bi_arr = ak.bigint_from_uint_arrays([bits, bits, bits, bits])
+        res = ak.ones_like(bi_arr)
+        bi_arr[:] = res
+        assert bi_arr.to_list() == res.to_list()
+
+
+def gather_scatter(a):
```
Comment (jaketrookman) [inline PROTO_tests/tests/bigint_agg_test.py:55]: Does this need a `@staticmethod` or is it ok because it is outside of the Class indentation

**Review by Ethan-DeBandi99 — COMMENTED** [review]
Overall, looks good. Added one recommendation for moving a function, but otherwise agree with previous comments and the changes made on those fronts.

File: PROTO_tests/tests/bigint_agg_test.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import arkouda as ak
+import numpy as np
+
+import pytest
```
Comment (pierce314159) [inline PROTO_tests/tests/bigint_agg_test.py:4]: same deal with running `isort`

File: PROTO_tests/tests/bigint_agg_test.py (hunk 1 above)
Comment (pierce314159) [reply PROTO_tests/tests/bigint_agg_test.py:55]: i agree with ethan here

**Review by pierce314159 — APPROVED** [review]
Couple small things, but this looks mostly good to go to me. Nice work!

File: PROTO_tests/tests/bigint_agg_test.py (hunk 1 above)
Comment (TheOddCzar789) [reply PROTO_tests/tests/bigint_agg_test.py]: Moved.

File: PROTO_tests/tests/bigint_agg_test.py (hunk 2 above)
Comment (TheOddCzar789) [reply PROTO_tests/tests/bigint_agg_test.py]: Done

**Review by jaketrookman — APPROVED** [review]
Looks good

**Review by pierce314159 — APPROVED** [review]
Looks good!

**Pull request merged**